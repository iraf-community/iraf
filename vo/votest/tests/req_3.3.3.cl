#
#  Refer to a Resource by an 'anonymous' query.

# Set the test description string.
votest.descr = "Refer to a Resource by an 'anonymous' query."

print ("------------------------------------------------------------------")
print ("Req  3.3.3:  Users shall be able to refer to a VO Resource by the")
print ("	     'anonymous' result of a Registry query.")
print ("------------------------------------------------------------------")

fcache init


# Execute the test commands.
print ("")
print ("Process requests for all x-ray resources in our personal database. ")
print ("")
print ("")

# Search for the X-Ray resource.
regdb ("resolve", bandpass="x-ray")

if (regdb.svctype == "I") {
    getimg (regdb.alias, "dev$ypix", size=0.02)
} else if (regdb.svctype == "C") {
    getcat (regdb.alias, "dev$ypix", size=0.02)
} else {
    printf ("Unknown service type for '%s'\n", regdb.alias)
}
