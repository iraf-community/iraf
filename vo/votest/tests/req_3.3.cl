#
#  Registry query in multiple ways.

# Set the test description string.
votest.descr = "Registry query in multiple ways."

print ("------------------------------------------------------------------")
print ("Req  3.3:  Users shall be able to refer to a VO Resource (in tasks")
print ("           requiring a Resource parameter) in multiple ways:")
print ("------------------------------------------------------------------")


fcache init


# Execute the test commands.
print ("This requirement is demonstrated by the sub-tests.\n")
