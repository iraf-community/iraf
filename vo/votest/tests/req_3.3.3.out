------------------------------------------------------------------
Req  3.3.3:  Users shall be able to refer to a VO Resource by the
	     'anonymous' result of a Registry query.
------------------------------------------------------------------

Process requests for all x-ray resources in our personal database. 


# Survey Ra Dec Dim Size Scale Format PixFlags URL LogicalName
rass3bb 202.4702333333333 47.19455277777778 2 "300 300" "-3.6287224658266664E-4 3.6287224658266664E-4" image/fits F http://skyview.gsfc.nasa.gov/cgi-bin/images?position=202.4702333333333%2C47.19455277777778&survey=rass3bb&pixels=300%2C300&sampler=Clip&size=0.1088616739748%2C0.1088616739748&projection=Tan&coordinates=J2000.0&return=FITS 1
rass3bb 202.4702333333333 47.19455277777778 2 "300 300" "-3.6287224658266664E-4 3.6287224658266664E-4" image/jpeg F http://skyview.gsfc.nasa.gov/cgi-bin/images?position=202.4702333333333%2C47.19455277777778&survey=rass3bb&pixels=300%2C300&sampler=Clip&size=0.1088616739748%2C0.1088616739748&projection=Tan&coordinates=J2000.0&nofits=1&quicklook=jpeg&return=jpeg 1
rass3hb 202.4702333333333 47.19455277777778 2 "300 300" "-3.6287224658266664E-4 3.6287224658266664E-4" image/fits F http://skyview.gsfc.nasa.gov/cgi-bin/images?position=202.4702333333333%2C47.19455277777778&survey=rass3hb&pixels=300%2C300&sampler=Clip&size=0.1088616739748%2C0.1088616739748&projection=Tan&coordinates=J2000.0&return=FITS 2
rass3hb 202.4702333333333 47.19455277777778 2 "300 300" "-3.6287224658266664E-4 3.6287224658266664E-4" image/jpeg F http://skyview.gsfc.nasa.gov/cgi-bin/images?position=202.4702333333333%2C47.19455277777778&survey=rass3hb&pixels=300%2C300&sampler=Clip&size=0.1088616739748%2C0.1088616739748&projection=Tan&coordinates=J2000.0&nofits=1&quicklook=jpeg&return=jpeg 2
rass3sb 202.4702333333333 47.19455277777778 2 "300 300" "-3.6287224658266664E-4 3.6287224658266664E-4" image/fits F http://skyview.gsfc.nasa.gov/cgi-bin/images?position=202.4702333333333%2C47.19455277777778&survey=rass3sb&pixels=300%2C300&sampler=Clip&size=0.1088616739748%2C0.1088616739748&projection=Tan&coordinates=J2000.0&return=FITS 3
rass3sb 202.4702333333333 47.19455277777778 2 "300 300" "-3.6287224658266664E-4 3.6287224658266664E-4" image/jpeg F http://skyview.gsfc.nasa.gov/cgi-bin/images?position=202.4702333333333%2C47.19455277777778&survey=rass3sb&pixels=300%2C300&sampler=Clip&size=0.1088616739748%2C0.1088616739748&projection=Tan&coordinates=J2000.0&nofits=1&quicklook=jpeg&return=jpeg 3
